frontend/src/components/common/category/helpers/helpers.ts

```diff
@@ -1 +1 @@
-export { getRandomColor } from './get-random-color.helper';
+export { getRandomColorClassName } from './get-random-color.helper';
```

Comment:
```suggestion
export { getRandomColorClassName } from './get-random-color/get-random-color.helper';
```
each helper should be in the separate folder